sensor: pms7003: Convert to DEVICE_DT_GET

galak opened the pull request:
Move to using DEVICE_DT_GET so we can phase out DT_LABEL.

Signed-off-by: Kumar Gala <galak@kernel.org>

galak committed:
sensor: pms7003: Convert to DEVICE_DT_GET

Move to using DEVICE_DT_GET so we can phase out DT_LABEL.

Signed-off-by: Kumar Gala <galak@kernel.org>

gmarull commented on drivers/sensor/pms7003/pms7003.c:
```diff
@@ -172,12 +172,11 @@ static int pms7003_init(const struct device *dev)
 
-	if (!drv_data->uart_dev) {
-		LOG_DBG("uart device is not found: %s",
-			    DT_INST_BUS_LABEL(0));
-		return -EINVAL;
+	if (!device_is_ready(uart->dev)) {
+		LOG_ERR("Bus device is not ready");
+		return -ENODEV;
 	}
```
device can now be placed in driver's config and initialized statically.

galak committed:
tests: build_all: uart: add pms7003

There was no test building the PMS7003 sensor driver, so add it
to build_all.

Signed-off-by: Kumar Gala <galak@kernel.org>

galak committed:
sensor: pms7003: Fix compile errors with driver

We get type mismatch errors because of use of K_MSEC.  As the timeout
is defined in terms of milliseconds and k_uptime_get() returns
milliseconds we can just remove K_MSEC usage.

Signed-off-by: Kumar Gala <galak@kernel.org>
